gui/pulsed/pulsed_maingui.py

```diff
@@ -206,6 +207,36 @@ def on_activate(self):
         self._connect_logic_signals()
 
         self.show()
+
+        if np.isnan(self.pulsedmasterlogic().sequencegeneratorlogic().get_speed_write_load()):
+            dialog = QtWidgets.QDialog(self._mw)
```

Comment:
You are creating a `QWidget` object here and giving it a parent `QObject` (`self._mw`).
If you are not explicitly destroying this dialog object after use, a reference will remain in the child tree of `self._mw` and it will not get garbage collected.
I see 2 ways to solve this:

1) explicitly set the parent to `None` at the end of this method

yet better

2) Use [`QMessageBox`](https://doc.qt.io/qt-5/qmessagebox.html#details) instead (also saves a lot of unnecessary code that is becoming obsolete in the new qudi core)

Reply:
Switched to QMessageBox. Minor disadvantage: as far as I see, there is no way to center the button position. This seems to be style for other qudi dialogues. Now it looks like:
![image](https://user-images.githubusercontent.com/5861249/107119842-6ab5b180-688a-11eb-8ce1-3785873e41ff.png)